ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v0.18.4-periodics.yaml

```diff
@@ -15,6 +15,14 @@ periodics:
     ci-operator.openshift.io/variant: "46"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-knative-eventing-release-v0.18.4-46-e2e-aws-ocp-46-continuous
+  reporter_config:
+    slack:
+      channel: '#knative-eventing'
+      job_states_to_report:
+      - success
+      - failure
+      - error
+      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}} :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :volcano: {{end}}'
```

Comment:
Was this forgotten for this job?

Reply:
yep. I wanted to add that only once replace the 0.18.3 job w this.

Avoiding too much noise